controllers/storagecluster/reconcile.go

```diff
@@ -114,7 +114,7 @@ var validTopologyLabelKeys = []string{
 // +kubebuilder:rbac:groups=core,resources=namespaces,verbs=get
 // +kubebuilder:rbac:groups=apps,resources=deployments;daemonsets;replicasets;statefulsets,verbs=get;list;watch;create;update;delete
 // +kubebuilder:rbac:groups=monitoring.coreos.com,resources=servicemonitors;prometheusrules,verbs=get;list;watch;create;update;delete
-// +kubebuilder:rbac:groups=snapshot.storage.k8s.io,resources=volumesnapshots;volumesnapshotclasses,verbs=get;create;delete
+// +kubebuilder:rbac:groups=snapshot.storage.k8s.io,resources=volumesnapshots;volumesnapshotclasses,verbs=get;watch;create;delete
```

Comment:
AFAIK ocs-operator doesnt need volumesnapshots permission that can be removed and also you need update access for volumesnapshotclass

Reply:
https://github.com/red-hat-storage/ocs-operator/pull/2774#issuecomment-2325063959
@Madhu-1 I don't find the `update` verb in the code.
Removed `volumesnapshots` resource

Reply:
https://github.com/red-hat-storage/ocs-operator/blob/main/controllers/storagecluster/volumesnapshotterclasses.go#L132 here we are doing update

Reply:
@Madhu-1 @iamniting done